### PR TITLE
PR for DUDes and kaiju recipes

### PR DESCRIPTION
**Pull request by marcelmat12**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core

### CODE REVIEW

File: recipes/kaiju/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package:
+  name: kaiju
+  version: "1.0"
+
+source:
+  git_url: https://github.com/bioinformatics-centre/kaiju.git
+  
+requirements:
+  build:
+    - gcc
```
Comment (bgruening) [inline recipes/kaiju/meta.yaml]: Can you please add llvm for OSX as descriped in our guideline?

**Comment by bgruening** [comment]
Looks wonderful, please fix the llvm part and we are ready to go :)

**Comment by marcelmat12** [comment]
Done as requested :) 
Thanks for the review

**Comment by marcelmat12** [comment]
Sorry but had to skip the build on OSX, 
im not so familiar with OSX and always get a compile error there.

**Comment by bgruening** [comment]
thanks @marcelmat12 !

**Comment by daler** [comment]
@marcelmat12 @bgruening doens't look like DUDes made it into anaconda. Testing the recipe building now gives an md5sum mismatch. Did something change in the source recently?

**Comment by marcelmat12** [comment]
@daler It appears that something changed
the md5 for DUDes is now:
dde2c8dbca04448cf21a0ed6b092b047
and the sha1:
7de64f04a48f895354d7c0a4cd7c9d1f7d1edc5a

Im unfortunately on my phone at the moment and wont be able to change until tomorrow

**Comment by marcelmat12** [comment]
Thanks @bgruening ! 

Appreciate your help with my first PR :)

**Pull request merged**